song-server/src/main/java/org/icgc/dcc/song/server/config/SecurityConfig.java

```diff
@@ -34,7 +34,7 @@
 @EnableResourceServer
 public class SecurityConfig extends ResourceServerConfigurerAdapter {
 
-  @Value("${auth.server.uploadScope}")
+  @Value("${auth.server.suffix}")
   private String uploadScope;
 
```

Comment:
`song.<studyId>.<suffix>` That was you can run multiple song servers with segregated data, scope schemes.